content/tracking/tracking_3.md

```diff
@@ -0,0 +1,98 @@
++++
+title = "Adding Context"
+weight = 3
+post = ""
++++
+
+Whilst the tracking set up provides event data on user **actions** at a specific point in time, **context** describes the setting in which an event takes place. To describe the context of an event, we need to define and capture individual **entities**. For example:
+
+- The user performing an action
+- The web page the action occured on
+- A product that has been interacted with
+
+Together, these entities make up the context of an event.
+
+Similar to the predefined events, a number of entities are available to implement out of the box including:
+
+- **webPage Entity -** Adds the Pageview ID 
+- **session Entity -** Information about the user session
+- **performanceTiming Entity -** Calculate page performance metrics
+- **geolocation Entity -** Information on the users location
+
+{{< tabs groupId="select" >}}
+{{% tab name="JS" %}}
+
+#### **Step 1:** Enable Predefined Entities
```

Comment:
Change "Predefined Entities" to be all lower case to be consistent with other titles